rhea/cores/uart/uartbase.py

```diff
@@ -62,7 +62,24 @@ def rtlbaud():
 
 
 def uarttx(glbl, fbustx, tx, baudce):
-    """
+    """UART transmitter  function
+       
+    Ports
```

Comment:
This isn't the "Google style" docs, http://sphinxcontrib-napoleon.readthedocs.org/en/latest/index.html#google-vs-numpy

I know I haven't been consistent in the docs either, I think moving to the Google style is the correct direction.  We need to use
```
Arguments (ports):
```
and see if sphinx picks up on it, otherwise will have to leave the `(ports)` out.